osdep/language-posix.c

```diff
@@ -38,8 +38,13 @@ char **mp_get_user_langs(void)
     // Prefer anything we get from LANGUAGE first
     for (const char *langList = getenv("LANGUAGE"); langList && *langList;) {
         size_t len = strcspn(langList, ":");
+        char *lang = talloc_strndup(ret, langList, len);
```

Comment:
`MP_TARRAY_GROW(NULL, ret, nb);` has to be above this line, else `ret` will be null and this will be potentially leaked string.